pages/index.js

```diff
@@ -239,39 +267,10 @@ export default function Home({
         </div>
       </div>
 
-      <div className="bg-primary-low dark:drop-shadow-none dark:bg-dark mb-8 p-8 drop-shadow-md">
-        {alerts.map((alert, index) => (
-          <Alert key={index} type={alert.type} message={alert.message} />
-        ))}
```

Comment:
Thank you for your contribution 👍  

But only this alert was meant to be moved, not the statistics...

<img width="1672" alt="Screenshot 2023-09-09 at 04 23 07" src="https://github.com/EddieHubCommunity/BioDrop/assets/624760/e5c84193-3f27-44fa-834b-236d9ce8cc83">